### DISCUSSION TITLE
Deceptive Site Ahead flags from Google

### DISCUSSION THREAD

**Discussion by y3owk1n** [nextauthjs/next-auth#8176] [Help]
### Reproduction URL

https://student.skbadmintonclub.com

### Describe the issue

We're facing this issue in production, where all browsers are showing Deceptive site ahead on all endpoints related to NextAuth. I have read about a previous issue #6393 and it was no help, unfortunately.

I'm using NextAuth + Google Oauth and it was a pretty simple setup as well.

The following is my setup, pretty basic.

```js
    const authOptions: NextAuthOptions = {
        // Configure one or more authentication providers
        adapter: PrismaAdapter(prisma),
        providers: [
            GoogleProvider({
                clientId: process.env.GOOGLE_CLIENT_ID as string,
                clientSecret: process.env.GOOGLE_CLIENT_SECRET as string,
            }),
            // ...add more providers here
        ],
        secret: process.env.NEXTAUTH_SECRET,
       ... rest,
```

The issue came and caused our user scare because Red screen is being shown.

Here's the result for one of the nextauth endpoint on Safe Browsing Checker.

<img width="1156" alt="image" src="https://github.com/nextauthjs/next-auth/assets/62775956/ccc2e1f9-e65b-45dc-af56-b64b26a2d58e">

And this is the result from Google Search Console.

<img width="1156" alt="image" src="https://github.com/nextauthjs/next-auth/assets/62775956/16148a98-5e55-4aa0-b65a-f50c34a613ba">

Might be some config or something else that I had missed if it's just only me having the issue, appreciate if able to get any help from here.

### How to reproduce

Simply access any of our subdomain, they are sharing the same config.

- https://student.skbadmintonclub.com
- https://coach.skbadmintonclub.com

### Expected behavior

Google should not flag it as phishing site.

**Comment by ghjunior** [comment]
Are you still running into this issue? Just deployed a brand new app and getting the same flag from Google.

**Comment by vhpoet** [comment]
I'm getting this too. New domain, pristine setup, no weird configs.

<img width="719" alt="Screenshot 2023-09-24 at 11 10 06 PM" src="https://github.com/nextauthjs/next-auth/assets/686185/a0dad619-264c-4a54-9f8c-8a3bef74de9c">
